docs/pages/web/slimbanner.js

```diff
@@ -445,23 +444,46 @@ Combine SlimBanners with other components like [Callouts](/web/callout) or [Upse
           />
         </MainSection.Subsection>
         <MainSection.Subsection
-          description="The SlimBanner message can be complemented with a helper Link."
-          title="helperLink"
+          description={`The \`message\` prop accepts both strings and [Text](/Text). Use strings for simple messages without any visual style. SlimBanner will handle the message style and the adherence to design guidelines. If a message with more complex style is required, such as bold text or inline links, use Text to inline text and links within the same Text component.
```

Comment:
"…prop accepts _either a string or_ Text." Current wording makes it sound like the user could pass an array of strings. You'll need to adjust to the singular form in the next sentence as well.

No "the" needed in "…will handle the message style _and adherence_ to…"

Last sentence is kinda awkward: "…such as bold text or inline links, use Text to wrap your message with any additional Text or Link usages contained within." (or something like that)

Reply:
<img width="984" alt="Screen Shot 2022-11-10 at 11 57 58 AM" src="https://user-images.githubusercontent.com/10593890/201158786-49db472b-f0e3-4b5c-bc12-536b1296c8a8.png">
<img width="626" alt="Screen Shot 2022-11-10 at 11 58 30 AM" src="https://user-images.githubusercontent.com/10593890/201158788-a50dc40c-3753-41a1-abf3-d26de2871c45.png">

Reply:
.name doesn't work. I have to add the following to get the displayName available

```
Text.displayName = 'Text';

export default Text;
```

Reply:
DONE

Reply:
I'm not seeing the updated `description` here?

Reply:
Mmmm... let me double check

Reply:
I'm still not seeing updates to the copy:

> "…prop accepts either a string or Text." Current wording makes it sound like the user could pass an array of strings. You'll need to adjust to the singular form in the next sentence as well.
>
> No "the" needed in "…will handle the message style and adherence to…"
>
> Last sentence is kinda awkward: "…such as bold text or inline links, use Text to wrap your message with any additional Text or Link usages contained within." (or something like that)

Reply:
DONE